main.js

```diff
@@ -16,51 +16,55 @@ function generateId() {
 }
 
 function storeData() {
-  localStorage.setItem('books', JSON.stringify(bookList));
+  localStorage.setItem('bookList', JSON.stringify(bookList));
 }
```

Comment:
- [ ] please as the requirements you should also refactor this function into class so it should a method's class I encourage you to do that